django_irods/storage.py

```diff
@@ -308,9 +308,58 @@ def listdir(self, path):
                     listing[2].append(size)
         return listing
 
+    def listdir(self, path):
+        # the query below returns name and size (separated in comma) of all data
+        # objects/files under the path collection/directory
+        qrystr = "select DATA_NAME, DATA_SIZE where DATA_REPL_STATUS != '0' AND " \
+                 "COLL_NAME like '%{}'".format(path)
+        stdout = self.session.run("iquest", None, "--no-page", "%s,%s",
```

Comment:
iquest... this is a new command for me.  I've never heard of this.  It looks like you're accessing the iRODS database?  Could you point me to some resources so I can understand this command?

Reply:
@sblack-usu Here is the doc for iquest: https://docs.irods.org/4.2.0/icommands/user/#iquest